apps/dashboard/app/views/batch_connect/sessions/connections/_native_vnc.html.erb

```diff
@@ -1,39 +1,56 @@
+<%# Random port from 10000 - 65535 deterministically generated %>
+<% localport = Digest::MD5.hexdigest("#{connect.host}:#{connect.port}").to_i(16) % 55535 + 10000 %>
+
 <% if browser.platform == :windows %>
 
   <ol>
     <li>
-      Download the latest
-      <a href="https://github.com/OSC/osc-connect/releases/latest" rel="noopener" target="_blank">OSC Connect</a>
-      if you don't already have it
+      Download any VNC viewer,
+      <a href="https://www.realvnc.com/en/connect/download/viewer/" rel="noopener" target="_blank">RealVNC</a>
+      is a good option.
     </li>
     <li>
-      Launch <strong>OSC Connect</strong>
+      Start an SSH tunnel through your preferred SSH client or through your <a href="https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701">Windows Terminal</a> Linux distribution.
     </li>
     <li>
-      Click - <%= link_to "osc://:#{connect.password}@#{connect.host}:#{connect.port}", "osc://:#{connect.password}@#{connect.host}:#{connect.port}", onclick: "$.get('" + analytics_path(type: "launch_osc_connect") + "');" %>
+      <% if ENV["OOD_NATIVE_VNC_LOGIN_HOST"] %>
```

Comment:
Does this add a second environment variable? We already have `ENABLE_NATIVE_VNC` in `app/views/batch_connect/sessions/_panel.html.erb`. 

Maybe this is an opportunity to put that into the configuration singleton and alias it with `OOD_ENABLE_NATIVE_VNC`?

Reply:
OOD_ENABLE_NATIVE_VNC just enables it. OOD_NATIVE_VNC_LOGIN_HOST is if you want to specify the host to use. I guess if you are going to specify the host to use, its kinda redundant to also have to enable by specifying OOD_ENABLE_NATIVE_VNC...

Reply:
I could see an argument for aliasing ENABLE_NATIVE_VNC to OOD_ENABLE_NATIVE_VNC since we are moving to prepending OOD to ENV vars now